### ISSUE TITLE
Example using_kinematic2d.zip is still Godot 3.x.

### ISSUE THREAD

**Issue by Vojaz** [godotengine/godot-docs#7087]
**Godot 4.0**

The example _using_kinematic2d.zip_ seems to be still Godot 3.x.

https://docs.godotengine.org/en/stable/tutorials/physics/using_character_body_2d.html

**Comment by jynus** [comment]
Hi,

I started working on updating the example files for this tutorial. In most cases it is just minor API updates on KinematicBody -> CharacterBody that the automatic updater cannot detect.

However, I have found that because the walls were at exactly 45º and default sliding angle is 44.9999 degrees (due to radian conversion), there is some non-trivial artifact that may confuse beginners. While this would be easy to overcome with an extra configuration in a real case, given this is a tutorial, I am trying to update the code to be as simple as possible and useful for the readers of the tutorial instructions.

There is arguably also some learnings here for setting non-confusing default values on code (to be seen).
